crates/oxc_linter/src/snapshots/for_direction.snap

```diff
@@ -0,0 +1,54 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: for_direction
+---
+
+  ⚠ eslint(for-direction): Disallow for loop in the wrong direction
+   ╭─[for_direction.tsx:1:1]
+ 1 │ for (var a = 0; a < 10; a--) {}
+   · ───────────────────────────────
+   ╰────
+  help: Use while loop for intended infinite loop
```

Comment:
```
  ⚠ eslint(for-direction): The update clause in this loop moves the variable in the wrong direction
   ╭─[for_direction.tsx:1:16]
 1 │ for(var i = 0; i < 10; i--){}
   ·                ───┬──  ─┬─
   ·                   │     ╰── with this update
   ·                   ╰── This test moves in the wrong direction
   ╰────
```

I think I need to write up a guide on what to point at, but basically we want to minimize the span to point at for context.

Reply:
Fixed.